src/ralph/lib/hooks.py

```diff
@@ -0,0 +1,21 @@
+import pkg_resources
+
+from django.conf import settings
+from django.utils import lru_cache
+
+
+@lru_cache.lru_cache(maxsize=None)
+def get_hook(module_name, name, variant=None):
```

Comment:
IMO it would be nice to put here docstring documenting how to use this function and how to setup it in settings.